app/index.html

```diff
@@ -0,0 +1,640 @@
+                            <a href="" class="social-icon"><i class="mdi mdi-google-glass"></i> </a>
+                            <a href="" class="social-icon"><i class="mdi mdi-heart"></i> </a>
+                        </div>
+                    </div>
+                </div>
+
+
+                <div class="col-md-offset-1 col-md-6 col-sm-11 col-xs-12">
+                    <div class="row">
+                        <form>
+                            <div class="form-group col-sm-6">
+                                <input type="text" placeholder="Name">
+                            </div>
+                            <div class="form-group col-sm-6">
+                                <input type="text" placeholder="Email">
+                            </div>
+
+                            <div class="form-group col-sm-12">
+                                <input type="text" placeholder="Subject">
+                            </div>
+
+                            <div class="form-group col-sm-12 pd-top-1">
+                                <textarea placeholder="Messages" rows="5"></textarea>
+                            </div>
+
+                            <div class="col-sm-12">
+                                <button type="submit" class="btn btn-submit">Send Message</button>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+
+    <section id="footer" class="pd-top-5 pd-bottom-4 footer">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-6 col-xs-12 text-left">
+                    <p>Copyright © <a href>Shin</a> 2016. All rights reserved</p>
+                </div>
+
+                <div class="col-md-offset-2 col-md-4 col-xs-12 col-xs-offset-0 text-right">
+                    <div class="social inline">
+                        <a href="" class="social-icon"><i class="mdi mdi-facebook"></i> </a>
+                        <a href="" class="social-icon"><i class="mdi mdi-twitter"></i> </a>
+                        <a href="" class="social-icon"><i class="mdi mdi-google-glass"></i> </a>
+                        <a href="" class="social-icon"><i class="mdi mdi-heart"></i> </a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+   
+
+
+</div>
+
+<script src="bower_components/jquery/dist/jquery.js"></script>
+<script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
+<script src="http://cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.2/modernizr.js"></script>
+<script src="assets/js/html5lightbox/html5lightbox.js"></script>
+<script src="bower_components/owl.carousel/dist/owl.carousel.min.js"></script>
+<script src="assets/js/main.js"></script>
+<script src="assets/js/nav.js"></script>
+<script src="bower_components/isotope/dist/isotope.pkgd.min.js"></script>
```

Comment:
first of all you have to include not your libraries. after it you can include your files.